visidata/loaders/unzip_http.py

```diff
@@ -149,7 +149,13 @@ def namelist(self):
         return list(r.filename for r in self.infoiter())
 
     def infoiter(self):
-        resp = self.http.request('HEAD', self.url)
+        urllib3 = vd.importExternal('urllib3')
```

Comment:
This file (unzip_http.py) should be taken directly from https://github.com/saulpw/unzip-http/blob/master/unzip_http.py, with the `vd.importExternal` line being manually substituted.  It's not a clean process but I didn't/don't expect unzip_http to be updated that often.  They've already drifted a bit, but if we could bring them into sync again and make these changes in a way that keeps them as close as possible, that's my preference.  Probably the easiest way is to let these errors percolate up and then handle them at a visidata layer instead of in unzip_http.